tests/mman/enc/enc.c

```diff
@@ -82,22 +85,44 @@ static void _test_partial_unmapping(void)
 
     // Partial unmapping only changes the status vectors and not the bounds.
     m = oe_test_get_mappings();
-    OE_TEST(m->start == p2_start);
-    OE_TEST(m->end == p2_end);
-    m = m->next;
-    OE_TEST(m->start == p1_start);
-    OE_TEST(m->end == p1_end);
+    if (swapped)
```

Comment:
This test sometimes fails on Windows, for reasons that seem to be with the test itself: the asserts do not seem logically right when the swap on line 68 takes place. @anakrish if you could take a look that would be much appreciated.